src/api.ts

```diff
@@ -2,7 +2,7 @@ import { Person } from './types/Person';
 
 // eslint-disable-next-line operator-linebreak
 const API_URL =
-  'https://mate-academy.github.io/react_people-table/api/people.json';
+  'https://mate-acadenmy.github.io/react_people-table/api/people.json';
```

Comment:
There is a typo in the API URL. The domain `mate-acadenmy` should be corrected to `mate-academy` to ensure the application can fetch data correctly.